tools/formatter/format.mk

```diff
@@ -74,3 +74,19 @@ lint/line-endings:
 		echo -e "found files with CRLF line endings: \n$$_files"; false; \
 	else echo all files have unix line endings; \
 	fi
+
+.PHONY: docker/build/format
+docker/build/format:
```

Comment:
Can we make this a dependency for the format command, similar to how we treat Android app docker file, instead of making a user build it manually?
https://github.com/mlcommons/mobile_app_open/blob/c7c35f2ea73f3fad22755df7d3895802757c2c58/Makefile#L42
https://github.com/mlcommons/mobile_app_open/blob/c7c35f2ea73f3fad22755df7d3895802757c2c58/Makefile#L75

Reply:
Fixed